streamwebs_frontend/streamwebs/forms.py

```diff
@@ -38,3 +39,59 @@ class Meta:
             'school': _('School'),
             'birthdate': _('Birthdate'),
         }
+
+
+class WQForm(forms.ModelForm):
+    notes = forms.CharField(required=False)
```

Comment:
Tests look great, but I don't think explicitly setting form fields to be optional is necessary, since these are model forms and the [optional fields were already declared to be blank=True in models.py](https://docs.djangoproject.com/en/1.9/topics/forms/modelforms/#field-types). Try commenting your explicit form fields out and see if your tests still pass.